jsconfig.eslint-base.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "checkJs": false,
+    "noEmit": true,
+    "downlevelIteration": true,
+    "strictNullChecks": true,
+    "moduleResolution": "node"
+  },
+  "include": []
```

Comment:
do we need to have three config files? I wonder if "src" could be the base and "full" extends from that. Would redundant globs slow things down?

Reply:
It has a big enough impact for me to prefer this approach.